cmd/server-main_test.go

```diff
@@ -219,9 +219,9 @@ func TestIsDistributedSetup(t *testing.T) {
 			disks  []string
 			result bool
 		}{
-			{[]string{`4.4.4.4:c:\mnt\disk1`, `4.4.4.4:c:\mnt\disk2`}, true},
-			{[]string{`4.4.4.4:c:\mnt\disk1`, `localhost:c:\mnt\disk2`}, true},
-			{[]string{`localhost:c:\mnt\disk1`, `localhost:c:\mnt\disk2`}, false},
+			{[]string{`http://4.4.4.4:80/c:\mnt\disk1`, `http://4.4.4.4:80/c:\mnt\disk2`}, true},
+			{[]string{`http://4.4.4.4:9000/c:\mnt\disk1`, `http://localhost:9000/c:\mnt\disk2`}, true},
+			{[]string{`http://localhost:9000/c:\mnt\disk1`, `http://localhost:9001/c:\mnt\disk2`}, false},
```

Comment:
isDistributedSetup would never receive that in the code flow. Since by the time it receives it already had url.URL.